src/transfer/util/npopto_correction.py

```diff
@@ -9,23 +9,23 @@
 Y_PITCH = 20
 NUMEL_IN_COL = 192
 
+npopto_electrode_xpos_arr = [X_OFFSET, X_OFFSET + X_PITCH] * NUMEL_IN_COL
```

Comment:
can we wrap this in a function?  helps keep the global namespace clean and gives us something to test.  I think `npopto_electrode_xpos` and `npopto_electrode_ypos` are all you need below, so:

```
NPOPTO_ELECTRODE_XPOS, NPOPTO_ELECTROD_YPOS = doit(...)
```

should do it?

Reply:
Yep, will do it now

Reply:
Can I leave the constant variables outside or should they also be defined inside the function?

Reply:
I would leave whatever variables you need to be global outside and nothing else.